pyproject.toml

```diff
@@ -104,6 +104,14 @@ select = [
     "B",            # flake8-bugbear
 ]
 
+[tool.pyright]
+include = ["src/wheel"]
+ignore = ["docs", "tests"]
+pythonVersion = "3.8"
+typeCheckingMode = "strict"
+
+reportUnnecessaryTypeIgnoreComment = "warning"
+
```

Comment:
I was  bit conflicted about this. It doesn't look wrong, but it may be premature to include.
```suggestion
```

Reply:
Fair enough. I only put this in to see everything that was flagged. Maybe a less harsh setting, like “standard” instead of “strict”, would be more appropriate?